### PR TITLE
#163897176 Heroku deployment

### PR DESCRIPTION
**Pull request by kdaglas**
### What does this PR do?
This PR deploys our application to heroku

### Description of Task to be completed?
- add the static.json file
- add the commands for running the deployment
- add the required engines that run node version
- add the yarn dependency

### How should this be manually tested?
- fetch the branch using `git fetch origin ch-heroku-deployment-163897176`
- check into the branch using `git checkout ch-heroku-deployment-163897176`
- run `npm install yarn && yarn install ` to install the dependencies
- run `yarn run build` to start the development server
- navigate to https://ah-frontend-xmen.herokuapp.com/

### What are the relevant pivotal tracker stories?
[#163897176](https://www.pivotaltracker.com/story/show/163897176)

### CODE REVIEW

**Review by MuhweziDeo — DISMISSED** [review]
LGTM

**Review by malfahad — DISMISSED** [review]

**Comment by armstrongsouljah** [comment]
@kdaglas how many endpoints do you have deployed? Can I only access `/`?

**Review by calvinpete — DISMISSED** [review]
LGTM

**Comment by kdaglas** [comment]
since there are no features currently deployed, it will only access / @armstrongsouljah

**Comment by armstrongsouljah** [comment]
> since there are no features currently deployed, it will only access / @armstrongsouljah

The routing config had implemented two dummy components to test routing. `/login` however still shows **Authors heaven...**

**Comment by kdaglas** [comment]
yes those are there but I set my heroku to automatically deploy develop branch so until my branch gets merged to develop, those dummy routes won't work, but they work if am deploying my branch @armstrongsouljah all I need to do is re-deploy

**Comment by armstrongsouljah** [comment]
> yes those are there but I set my heroku to automatically deploy develop branch so until my branch gets merged to develop, those dummy routes won't work, but they work if am deploying my branch @armstrongsouljah all I need to do is re-deploy

Makes sense. @kdaglas

**Review by armstrongsouljah — DISMISSED** [review]
Nice work @kdaglas,  I have checked the new deployment and the routes configured earlier are working as expected.

File: static.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "root": "dist",
+  "clean_urls": true,
+  "routes": {
+    "/**": "index.html"
+  }
+}
```
Comment (RrNn) [inline static.json]: @kdaglas , you forgot to add a new line here.

Comment (kdaglas) [reply static.json]: let me add it

**Pull request merged**